showcase/gotemplate.md

```diff
@@ -0,0 +1,13 @@
+---
+name: gotemplate.io
+logo_url: https://gotemplate.io/images/logo.png
```

Comment:
It seems like I am getting a 403 on this image request? 😯

Reply:
I think it should work now.